apps/wallet/src/ui/app/pages/home/transfer-coin/index.tsx

```diff
@@ -109,61 +113,52 @@ function TransferCoinPage() {
             maxSuiSingleCoinBalance,
         ]
     );
-    const validationSchemaStepTwo = useMemo(
-        () => createValidationSchemaStepTwo(),
-        []
-    );
 
     const dispatch = useAppDispatch();
     const navigate = useNavigate();
-    const onHandleSubmit = useCallback(
-        async (
-            { to, amount }: FormValues,
-            { resetForm }: FormikHelpers<FormValues>
-        ) => {
-            if (coinType === null || !gasBudgetEstimationUnits) {
-                return;
-            }
-            setSendError(null);
-            trackEvent('TransferCoins', {
-                props: { coinType },
-            });
-            try {
-                const bigIntAmount = parseAmount(amount, coinDecimals);
-                const response = await dispatch(
-                    sendTokens({
-                        amount: bigIntAmount,
-                        recipientAddress: to,
-                        tokenTypeArg: coinType,
-                        gasBudget: gasBudgetEstimationUnits,
-                    })
-                ).unwrap();
-
-                resetForm();
-                const txDigest = getTransactionDigest(response);
-                const receiptUrl = `/receipt?txdigest=${encodeURIComponent(
-                    txDigest
-                )}&from=transactions`;
-
-                navigate(receiptUrl);
-            } catch (e) {
-                setSendError((e as SerializedError).message || null);
-            }
-        },
-        [dispatch, navigate, coinType, coinDecimals, gasBudgetEstimationUnits]
-    );
 
-    const handleNextStep = useCallback(
-        (_: FormValues, { setSubmitting }: FormikHelpers<FormValues>) => {
-            setCurrentStep((prev) => prev + 1);
-            setSubmitting(false);
-        },
-        []
-    );
+    const closeSendToken = useCallback(() => {
+        navigate('/');
+    }, [navigate]);
 
-    const handleBackStep = useCallback(() => {
-        setCurrentStep(DEFAULT_FORM_STEP);
-    }, []);
+    const onHandleSubmit = useCallback(async () => {
+        if (coinType === null || !gasBudgetEstimationUnits) {
+            return;
+        }
+
+        setSendError(null);
+        trackEvent('TransferCoins', {
+            props: { coinType },
+        });
+        try {
+            const bigIntAmount = parseAmount(formData.amount, coinDecimals);
+            //Todo:(Jibz) move to react-query
+            const response = await dispatch(
+                sendTokens({
+                    amount: bigIntAmount,
+                    recipientAddress: formData.to,
+                    tokenTypeArg: coinType,
+                    gasBudget: gasBudgetEstimationUnits,
```

Comment:
It seems we are adding max to the ui but for the case of paySuiAll we are not using it. Either we should hide/disable it or add complete the functionality otherwise we will have issues until we merge it with new PRs

Reply:
In the other PR https://github.com/MystenLabs/sui/pull/8353